### DISCUSSION TITLE
How to disable the default key mappings?

### DISCUSSION THREAD

**Discussion by junjiexh** [NvChad/NvChad#2962] [Q&A]
According to [docs](https://nvchad.com/docs/config/mappings), the way to disable a key mappings is `vim.keymap.del`. But it will disable my custom key mapping as well.

For example, I have a plugin which defines some key mapping during its start-up time:
```
    "christoomey/vim-tmux-navigator",
    cmd = {
      "TmuxNavigateLeft",
      "TmuxNavigateDown",
      "TmuxNavigateUp",
      "TmuxNavigateRight",
      "TmuxNavigatePrevious",
    },
    keys = {
      { "<c-h>", "<cmd><C-U>TmuxNavigateLeft<cr>" },
      { "<c-j>", "<cmd><C-U>TmuxNavigateDown<cr>" },
      { "<c-k>", "<cmd><C-U>TmuxNavigateUp<cr>" },
      { "<c-l>", "<cmd><C-U>TmuxNavigateRight<cr>" },
      { "<c-\\>", "<cmd><C-U>TmuxNavigatePrevious<cr>" },
    },

```
But when I type `verbose maps "<c-h>"`, i got :
```
n  <C-H>       * <C-W>h
                 switch window left
```
It seems like the default mapping override my custom mapping. Then i tried to disable the default mappings in `lua/mappings.lua` using `vim.keymap.del` just like this:
```
local nomap = vim.keymap.del
nomap("n", "<c-h>")
nomap("n", "<c-j>")
nomap("n", "<c-k>")
nomap("n", "<c-l>")
```
But it will disable my custom mappings as well. So if i type type `verbose maps "<c-h>"`, nothing shown.

After do some search on the Google, i found a [solution](https://github.com/NvChad/NvChad/issues/1984):
```
M.disabled = {
  n = {
    -- Disable to allow the plugin nvim-tmux-navigator to bind (see custom/plugins.lua)
    ["<C-h>"] = "",
    ["<C-l>"] = "",
    ["<C-j>"] = "",
    ["<C-k>"] = "",
  }
}
```
But since the Nvchad 2.5, the custom config structure has been removed. And if i try to add this to chadrc.lua, it does not take effect.

The only way i can figure out to set this key mapping correctly is to set it in a separate file like "lua/mappings.lua". But I really prefer the previous method. I am new to this plugin and very like it, any help is greatly appreciated! Thanks.

**Comment by siduck** [comment]
try noremap in your lazy keys field
